src/parsers/manifestjson.js

```diff
@@ -1,28 +1,38 @@
 import { PACKAGE_EXTENSION, VALID_MANIFEST_VERSION } from 'const';
 import log from 'logger';
-import { MANIFEST_VERSION_INVALID } from 'messages';
+import { MANIFEST_VERSION_INVALID, MANIFEST_NAME_INVALID } from 'messages';
```

Comment:
This is a nitpick, and possibly a thing we should clean up later, but can these be alphabetical? It makes stuff _way_ easier to scan for me that way.